### PR TITLE
[TESTMERGE] Brings Reebe back to the Clockcult mode

### PR DESCRIPTION
**Pull request by keronshb**
## About The Pull Request

This restores Reebe, allowing Clock Cultists to teleport back there using an Obelisk.  Aka reverts https://github.com/Citadel-Station-13/Citadel-Station-13/pull/13820

This does NOT make Reebe the starting area again, it just gives them the option to go there.

Tested locally, works without crashing.

## Why It's Good For The Game

Reebe was a nice prep area before the crashes.

## Changelog
:cl:
add: Readds Reebe
/:cl:

### CODE REVIEW

**Comment by BlueWildrose** [comment]
There was a very good reason why Reebe was removed from the clockcult gamemode, and it solely has to do with the clockcult speedrun meta we all remember and had to deal with at one point.

- destroy R&D servers
- destroy R&D consoles
- destroy ore silo
- destroy ORM
- steal guns
- "fuck you"

Bringing back Reebe, a HUB - whether or not it allowed you to teleport in the station from anywhere or not would invite at least a lite version of this, where they might steal stuff and other things to set up their base of operations. If they can just take away whatever they want from the station, they _will_ do it. 

From what I understand of you talking in Discord, you wanted to see if you could bring in the idea of "mandatory dungeons" as an event that requires that you enter them to deal with a threat to the station, having to shut something down within to put an end to the problems propagating on the station, as a form of new PVE content. Honestly, this is a great idea, but we don't need to bring back a clock cult feature that encourages the exploitation of this old meta to test this theory. Pretty sure if any players from the old days saw Reebe, they'd go back into that mindset again and immediately start a repeat of what happened before.

To put it simply, Reebe as a PVE area is a great idea, Reebe as a place for player clockies to indulge in a syndie sleeper among other things in peace isn't.

**Review by DeltaFire15 — APPROVED** [review]
Tested on serb, worked fine. Seems good to go.
